Misc/NEWS.d/next/Core and Builtins/2019-12-19-23-02-54.bpo-39091.9eCROl.rst

```diff
@@ -0,0 +1,2 @@
+Type check the result of constructing the new exception instance in
+:c:func:`_PyErr_CreateException`

```

Comment:
Since it is your first contribution, please add your name in Misc/ACKS.

You can also add "Patch by *yourname*." here. Don't forget about the ending period.